src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java

```diff
@@ -404,20 +405,21 @@ private void assertAddressIsTransformed(Map<String, Object> address, Map<String,
     }
 
     private void assertChildrenInfo(Map<String, Object> transformedCaseData) {
-        List<Map<String, Object>> children = (List)(transformedCaseData.get("childrenInfo"));
+        List<Map<String, ChildInfo>> children = (List) (transformedCaseData.get("childrenInfo"));
 
-        assertChild(children.get(0), asList("Johny Bravo", "2000-03-12", "male", "son", "SON","New Zeeland"));
-        assertChild(children.get(1), asList("Anne Shirley", "1895-03-12", "female", "daughter", "Daughter","Canada"));
+        ChildInfo child1 = children.get(0).get("value");
+        ChildInfo child2 = children.get(1).get("value");
+
+        assertChild(child1, asList("Johny Bravo", "2000-03-12", "male", "son", "SON", "New Zeeland"));
+        assertChild(child2, asList("Anne Shirley", "1895-03-12", "female", "daughter", "Daughter", "Canada"));
     }
 
-    private void assertChild(Map<String, Object> child, List<String> values) {
-        assertThat(child, allOf(
-                hasEntry("name", values.get(0)),
-                hasEntry("dateOfBirth", values.get(1)),
-                hasEntry("gender", values.get(2)),
-                hasEntry("relationshipToApplicant", values.get(3)),
-                hasEntry("relationshipToRespondent", values.get(4)),
-                hasEntry("countryOfResidence", values.get(5))
-        ));
+    private void assertChild(ChildInfo child, List<String> values) {
```

Comment:
Can we have a map in the second parameter here?

Reply:
I had it, but I couldn't use Immutable.of(v, k, v, ..., v, k) as it had only up to 5 pairs. The code then becomes less readable imo, but you know what you mean. I have tried to use map and decided to use a list as it's not a public interface and it's not very likely this would be re-used a lot.

What do others think?

Reply:
Or even better, a ChildInfo.

Reply:
ok

Reply:
Fair enough. I see what you mean.